src/components/JoinedList/JoinedList.jsx

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import './JoinedList.styles.scss';
+
+const JoinedList = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+
+  const activeRockets = rockets.filter((rocket) => rocket.reserve);
+  const activeMissions = missions.filter((mission) => mission.join);
```

Comment:
The `filter()` method is used to render the reserved rockets and missions 🚀 Fantastic